### ISSUE TITLE
Outdated ingress manifest for elastisys/influxdb

### ISSUE THREAD

**Issue by moule3053** [elastisys/compliantkubernetes-apps#839]
**Describe the bug**
The ingress manifes at https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/elastisys/influxdb/templates/ingress.yaml seems outdated. As a result, I get the following errors when deploying `compliantkubernetes-apps` on the service cluster.

```
Error: Failed to render chart: exit status 1: Error: unable to build kubernetes objects from release manifest: unable to recognize "": no matches for kind "Ingress" in version "networking.k8s.io/v1beta1"

Error: Failed to render chart: exit status 1: Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "serviceName" in io.k8s.api.networking.v1.IngressBackend, ValidationError(Ingress.spec.rules[0].http.paths[0].backend): unknown field "servicePort" in io.k8s.api.networking.v1.IngressBackend, ValidationError(Ingress.spec.rules[0].http.paths[0]): missing required field "pathType" in io.k8s.api.networking.v1.HTTPIngressPath]

Error: Failed to render chart: exit status 1: Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(Ingress.spec.rules[0].http.paths[0]): missing required field "pathType" in io.k8s.api.networking.v1.HTTPIngressPath
```

Error goes away after updating the manifest file according to https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource

**To Reproduce**
1. Run `./bin/ck8s apply sc`

**Expected behavior**
Deploy `compliantkubernetes-apps` on the service cluster without errors.

**Version (add all relevant versions):**
 - v0.19.0

**Comment by Xartos** [comment]
We don't support kubernete 1.22, which I assume that you are running.
This should be fixed in the next release (0.20), see [this PR](https://github.com/elastisys/compliantkubernetes-apps/pull/824)

**Comment by moule3053** [comment]
Oh I see. Yes, I'm running Kubernetes 1.22, that's why. Thanks!
